components/StarSearch/StarSearchChat.tsx

```diff
@@ -699,7 +699,10 @@ export function StarSearchChat({
             showCloseButton={!isMobile}
           />
         ) : null}
-        <div className="star-search relative -mt-1.5 flex flex-col px-2 justify-between items-center w-full h-full grow bg-slate-50">
+        <div
+          className="star-search relative -mt-1.5 flex flex-col px-2 justify-between items-center w-full h-full grow bg-slate-50"
+          data-is-embedded={embedded}
```

Comment:
Good old [data attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes) to the rescue.